openml/evaluations/evaluation.py

```diff
@@ -0,0 +1,18 @@
+
+class OpenMLEvaluation(object):
+
+    def __init__(self, run_id, task_id, setup_id, flow_id, flow_name,
```

Comment:
Could you please add a docstring describing the class. Otherwise one needs to guess what `value` and `array_data` are.